chapters/hpmor-chapter-013.tex

```diff
@@ -173,7 +196,8 @@ \chapter{Asking the Wrong Questions}
 Opportunities to do good are everywhere\\
 but darkness is where the light needs to be}
 \hbox{\scshape Cost of question: }&\hbox{\scshape 1 point}\\
-\hbox{\scshape Current points: }&\hbox{74}\\[1.5ex]
+\hbox{\scshape Current points: }&\hbox{74}\\%
```

Comment:
I don't understand this change.

Reply:
my bad, I had a script add linebreaks after \\
fixed in https://github.com/rrthomas/hpmor/pull/83